addons/website_blog/i18n/website_blog.pot

```diff
@@ -1154,6 +1154,16 @@ msgstr ""
 msgid "Leisure Facilities"
 msgstr ""
 
+#. module: website_blog
+#. odoo-python
+#: code:addons/website_blog/models/ir_qweb_fields.py:0
+#, python-format
+msgid ""
+"Limitation: manually specified content here is not translatable. Empty this "
+"area in the main language version of your website to fallback on the default"
+" behavior: showing the first (translated) sentences of your blog content."
```

Comment:
This is for your interested @pierre-pde, could you validate the wording and the UX flow? I kept it as simple as possible, we could have done nothing too but..

![image](https://github.com/odoo/odoo/assets/30048408/1162e165-72cf-4004-9eb6-a7818f5bfb1d)

Reply:
Replaced by `On your default language, empty the blog post description and save to get an automated (translated) summary.` after discussion